scapy/plist.py

```diff
@@ -652,13 +690,16 @@ def convert_to(self, other_cls, name=None, stats=None):
 
 
 class SndRcvList(PacketList):
-    __slots__ = []
+    __slots__ = []  # type: List[Any]
 
     def __init__(self, res=None, name="Results", stats=None):
+        # type: (Optional[Union[List[Packet], PacketList]], str, Optional[List[Packet]]) -> None  # noqa: E501
```

Comment:
Could you annotate the function parameters and use the construct `# type: (...) -> None` instead? This will remove the need of the `noqa` tag.